mempool/metrics.gen.go

```diff
@@ -40,28 +40,37 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Name:      "rejected_txs",
 			Help:      "Number of rejected transactions.",
 		}, labels).With(labelsAndValues...),
-		EvictedTxs: prometheus.NewCounterFrom(stdprometheus.CounterOpts{
-			Namespace: namespace,
-			Subsystem: MetricsSubsystem,
-			Name:      "evicted_txs",
-			Help:      "Number of evicted transactions.",
-		}, labels).With(labelsAndValues...),
 		RecheckTimes: prometheus.NewCounterFrom(stdprometheus.CounterOpts{
 			Namespace: namespace,
 			Subsystem: MetricsSubsystem,
 			Name:      "recheck_times",
 			Help:      "Number of times transactions are rechecked in the mempool.",
 		}, labels).With(labelsAndValues...),
+		AlreadyReceivedTxs: prometheus.NewCounterFrom(stdprometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "already_received_txs",
+			Help:      "Number of times transactions were received more than once.",
```

Comment:
```suggestion
			Help:      "Number of duplicate transaction reception (conservative estimate).",
```

Reply:
This is an auto generated file. Cannot make changes here.